File: Example/Java/Model/com/github/hebertialmeida/modelgen/Avatar.java
package com.github.hebertialmeida.modelgen;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import android.net.Uri;

public final class Avatar {

    final @Nullable Uri original;
    final @Nullable byte[] photo;
    final @Nullable Uri small;

    public Avatar(
            @Nullable Uri original,
            @Nullable byte[] photo,
            @Nullable Uri small
            ) {

        this.original = original;
        this.photo = photo;
        this.small = small;
    }

    @Nullable
    public Uri getOriginal() {
        return original;
    }

    @Nullable
    public byte[] getPhoto() {
        return photo;
    }

    @Nullable
    public Uri getSmall() {
        return small;
    }

}

File: Example/Kotlin/Model/com/github/hebertialmeida/modelgen/Avatar.kt
package com.github.hebertialmeida.modelgen

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize
import android.net.Uri

@Parcelize
data class Avatar(
    val original: Uri?,
    val photo: ByteArray?,
    val small: Uri?
): Parcelable
